fleeting_deploy_iso.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# FIXME(lranjbar): Look into a cleaner way to do this
+# Attach the fleeting.iso to the VM and start it
+virsh attach-disk ${CLUSTER_NAME}_fleeting_0 ${FLEETING_ISO_LOCATION} hdc --type cdrom --mode readonly
+virsh start ${CLUSTER_NAME}_fleeting_0 --force-boot
```

Comment:
I had to do [something similar](https://github.com/hardys/dev-scripts/blob/hypershift_wip/hypershift/03_hypershift_nodes.sh#L51..L62) for hypershift testing recently and ran into the problem that after install the VM rebooted from cdrom, not disk - does the AI workflow ensure the boot order here?

I'm not super happy with that WIP approach so happy to collaborate on a better solution which works for both cases :)

Reply:
AI makes changes to the boot order during install. However using a CDROM is kind of a known issue with it due to the how this standard is defined. Now that I think about it using a separate fleeting VM is a workaround by using a VM that is separate from the installation VMs for the cluster. You are right that it isn't following the customers flow exactly so I probably need to think on this a bit more.

I was thinking about going the route of ensuring we have the right libvirt domain XML for the VMs down the road. There is an [Ansible collection for libvirt](https://galaxy.ansible.com/community/libvirt) we can use directly for a lot of these operations but it requires the definitions to be in those libvirt domain XML. So I haven't started refactoring really but I'm thinking if we get the right domain XML we might be able to make this more smooth to use. Alot of what we get from the metal-dev-env is the domain XML so this is where I'm thinking I guess. 🤔 I apologize that this is a rambling thought I'm still working through the details.